nix/shared.nix

```
@@ -640,7 +640,7 @@ let
         pkgs.dockerTools.buildImage {
           inherit name;
 
-          contents = [ possibly-static."${name}" ];
+          contents = [ possibly-static."${name}", "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt" ];
```

https://hydra.dhall-lang.org/jobset/dhall-haskell/1493#tabs-errors suggests that the comma is wrong:

```suggestion
          contents = [ possibly-static."${name}" "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt" ];
```